components/org.wso2.carbon.identity.conditional.auth.functions.choreo/src/test/resources/org/wso2/carbon/identity/conditional/auth/functions/choreo/choreo-payload.json

```diff
@@ -0,0 +1,39 @@
+{
+  "stringKey": "stringValue",
```

Comment:
Don't we have to consider null values being sent in the payload?

Reply:
There's a limitation where null values are skipped when response is received and payload is sent which is not caused by any adaptive authentication mechanisms. Hence `Null` is skipped from the payload.
However the issue needs to identified and fixed.